it/downloads/index.md

```diff
@@ -42,26 +40,52 @@ Se hai problemi con la compilazione di Ruby considera l'utilizzo di uno
 degli strumenti di terze parti presenti nella sezione successiva. Potrebbero
 esserti di aiuto.
 
-* **Stabile Corrente:**
-  Ruby {{ site.data.downloads.stable[0] }}
+* **Versioni stabili:**{% for version in site.data.downloads.stable %}{% assign release = site.data.releases | where: "version", version | first %}
+  * [Ruby {{ release.version }}]({{ release.url.gz }})<br>
+    sha256: {{ release.sha256.gz }}{% endfor %}
+
+{% if site.data.downloads.preview %}
+* **Versioni di preview:**{% for version in site.data.downloads.preview %}{% assign release = site.data.releases | where: "version", version | first %}
+  * [Ruby {{ release.version }}]({{ release.url.gz }})<br>
+    sha256: {{ release.sha256.gz }}{% endfor %}
+{% endif %}
+
+{% if site.data.downloads.security_maintenance %}
+* **Nella fase di manutenzione di sicurezza (sono giunti a fine vita!):**{% for version in site.data.downloads.security_maintenance %}{% assign release = site.data.releases | where: "version", version | first %}
```

Comment:
- I'd call `fine vita` as `EOL` directly, it's a term that is understood by people without the need of being translated
- `(sono giunti a fine vita!)` that's not correct, they are not yet there, so it should be `raggiungeranno presto la loro EOL`

Reply:
Ok 👍